images/base-emu/Dockerfile

```diff
@@ -0,0 +1,105 @@
+ARG BASE_APP_IMAGE
+
+# hadolint ignore=DL3006
+FROM ${BASE_APP_IMAGE}
+
+ARG DEBIAN_FRONTEND=noninteractive
+# see: https://github.com/AppImage/AppImageKit/wiki/FUSE#docker
+ENV APPIMAGE_EXTRACT_AND_RUN=1
+
+#GitHub REST query versioning
+ARG GITHUB_REST_VERSION=2022-11-28
+
+# Install prereqs
+RUN \
+    echo "**** Install Common Prereqs (Mesa/Vulkan/Fuse/QT/Misc) ****" && \
+		apt-get update && \
+		apt-get install -y \
+            curl \
+            ffmpeg \
+            gpg-agent \
+            jq \
+            libdbus-1-3 \
+            libegl1 \
+            libglu1-mesa-dev \
+            libgtk-3-0 \
+            libsdl2-2.0-0 \
+            libvulkan1 \
+            p7zip-full \
+            python3-six \
+            qt5-qmake \
+            qtbase5-dev \
```

Comment:
Since it was removed from the dolphin install later, should we add `qtbase5-dev-tools` in here instead?

Reply:
Along with 
```
qt6-base-dev \
  qt6-base-dev-tools
```

Reply:
To be fair, I don't think we need any of those since we use the appimages which should bundle all required dependencies. I'm inclined to remove all those deps..

Reply:
Nope, tried it and I get errors looking for shared libraries; I guess we need those after all..